gem/lib/earlgrey/files/Swift-2.2/EarlGrey.swift

```diff
@@ -0,0 +1,88 @@
+//
+// Copyright 2016 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import EarlGrey
+
+public func grey_allOfMatchers(args: AnyObject...) -> GREYMatcher! {
+  return GREYAllOf.init(matchers: args)
+}
+
+public func grey_anyOfMatchers(args: AnyObject...) -> GREYMatcher! {
+  return GREYAnyOf.init(matchers: args)
+}
+
+public func EarlGrey() -> EarlGreyImpl! {
+  return EarlGreyImpl.invokedFromFile(__FILE__, lineNumber: __LINE__)
```

Comment:
shouldnt these be ##file  and ##line?
I am comparing with https://github.com/google/EarlGrey/blob/master/gem/lib/earlgrey/files/EarlGrey.swift